build/gulp/tasks/dist.ts

```diff
@@ -13,19 +13,31 @@ const { log, PluginError } = g.util
 // Clean
 // ----------------------------------------
 
-task('clean:dist', cb => {
-  rimraf(`${config.paths.dist()}/*`, cb)
+task('clean:dist:es', cb => {
+  rimraf(`${config.paths.dist()}/es/*`, cb)
 })
 
+task('clean:dist:commonjs', cb => {
+  rimraf(`${config.paths.dist()}/commonjs/*`, cb)
+})
+
+task('clean:dist:umd', cb => {
+  rimraf(`${config.paths.dist()}/umd/*`, cb)
+})
+
+task('clean:dist', parallel('clean:dist:es', 'clean:dist:commonjs', 'clean:dist:umd'))
+
 // ----------------------------------------
 // Build
 // ----------------------------------------
+const componentsSrc = [paths.src('**/*.{ts,tsx}'), `!${paths.src('**/umd.ts')}`]
```

Comment:
We were previously including the `umd` build with ES and CommonJS module types.  Now we exclude it.